src/Calibration.cpp

```diff
@@ -19,7 +16,7 @@ void Calibration::StartCalibration(vr::DriverPose_t maintainPose) {
     m_isCalibrating = true;
 }
 
-VRPoseConfiguration_t Calibration::FinishCalibration(vr::TrackedDevicePose_t controllerPose, VRPoseConfiguration_t poseConfiguration, bool isRightHand) {
+VRPoseConfiguration_t Calibration::StopCalibration(vr::TrackedDevicePose_t controllerPose, VRPoseConfiguration_t poseConfiguration, bool isRightHand) {
```

Comment:
In my opinion, `FinishCalibration` better describes what this method does, as rather than just stopping, it goes and recalculates the offsets.